daemon/api_notices.go

```diff
@@ -132,7 +132,7 @@ func getNotices(c *Command, r *http.Request, user *auth.UserState) Response {
 
 	if timeout != 0 {
 		// Wait up to timeout for notices matching given filter to occur
-		ctx, cancel := context.WithTimeout(r.Context(), timeout)
+		ctx, cancel := context.WithTimeout(c.d.tomb.Context(r.Context()), timeout)
```

Comment:
I think I'd add a comment that we're using daemon's tomb context so that the request will get canceled when the tomb gets killed when shutting down the daemon

Reply:
done